eggplant/membership/models/profile.py

```diff
@@ -33,8 +33,8 @@ def __str__(self):
     @property
     def full_name(self):
         "Returns member's full name."
-        if self.middlename:
-            return '{0} {1} {2}'.format(self.user.firstname, self.middlename,
+        if self.middle_name:
```

Comment:
It looks like it is still in UserProfile but was renamed from `middlename` to `middle_name`, it's just someone forgot to change it in this method.